web/src/context/ContextualLayers.svelte

```diff
@@ -15,7 +16,7 @@
   <div
     class="pico"
     style:background="grey"
-    style="display: flex; flex-direction: column"
+    style="display: flex; flex-direction: column; max-height: 80vh; overflow: auto"
```

Comment:
Ran out of vertical space. The NPW UI mocks have some ideas -- I'd be happy to implement any ideas you have for short-term improvements, but I'm still a bit holding out to borrow ideas from there.